paasta_tools/autoscaling/cluster_boost.py

```diff
@@ -13,8 +13,17 @@
 from datetime import datetime
 from time import time as get_time
 
+import staticconf
```

Comment:
I'm surprised this works. I don't see it in requirements.txt

Can we get away without it? We don't need all the fanciness that staticconf provides in this situation.

Reply:
clusterman_metrics reads all of its configuration from srv-configs via staticconf, so I don't think we can get away without it; it's included as a dependency of clusterman_metrics.  (It's also already in requirements.txt, `PyStaticConfiguration==0.10.3`)